### PR TITLE
include scipy for SE words

### PR DESCRIPTION
**Pull request by sophieball**

### CODE REVIEW

File: src/BUILD [hunk 1]
```diff
@@ -53,6 +53,7 @@ py_binary(
         ":fighting_words",
         requirement("langdetect"),
         requirement("pandas"),
+        requirement("scipy")
```
Comment (CaptainEmerson) [inline src/BUILD:56]: Still need joblib here

File: main/train_classifier_g.py [hunk 2]
```diff
@@ -120,7 +132,9 @@ def predict_unlabeled(unlabeled_data, trained_model, G_data=True):
     logging.info("Training the model and predicting labels.")
     [training, unlabeled] = receive_data.receive_data()
     trained_model = train_model(training)
-    predict_unlabeled(unlabeled, trained_model)
+    trained_model = train_model(training, model_name="rf")
+    trained_model = train_model(training, model_name="lg")
+    #predict_unlabeled(unlabeled, trained_model, feature_set[0])
```
Comment (CaptainEmerson) [inline main/train_classifier_g.py:137]: rm?

File: src/find_SE_words.py [hunk 3]
```diff
@@ -32,14 +32,23 @@ def preprocess_text(cur_text):
...
 
 review_comments = [preprocess_text(x) for x in reviews["text"].to_list()]
 other_comments = pd.read_csv("src/data/kaggle_toxicity_subset.csv")
-other_comments = other_comments.sample(len(review_comments))
+num_review = len(review_comments)
+num_kaggle = len(other_comments)
+if num_kaggle > num_review:
+  other_comments = other_comments.sample(num_review)
+elif num_review > num_kaggle:
+  review_comments = review_comments.sample(num_kaggle)
```
Comment (CaptainEmerson) [inline src/find_SE_words.py:51]:
```
Traceback (most recent call last):
  File "/usr/local/google/home/emersonm/toxicity-detector/bazel-bin/src/find_SE_words.runfiles/__main__/src/find_SE_words.py", line 51, in <module>
    review_comments = review_comments.sample(num_kaggle)
AttributeError: 'list' object has no attribute 'sample'
```

**Review by CaptainEmerson — CHANGES_REQUESTED** [review]

File: src/BUILD (hunk 1 above)
Comment (sophieball) [reply src/BUILD]: added

File: main/train_classifier_g.py (hunk 2 above)
Comment (sophieball) [reply main/train_classifier_g.py]: removed

File: src/find_SE_words.py (hunk 3 above)
Comment (sophieball) [reply src/find_SE_words.py]: fixed

**Comment by CaptainEmerson** [comment]
Seems to work for me. I get:
```
Vocab size is 14957
Comparing language...
```

But I'm not sure what this comment was for:
> Please move SE_words_GH_zscores.csv and SE_words_GH.list to src/data/

**Review by CaptainEmerson — APPROVED** [review]

**Comment by sophieball** [comment]
> Seems to work for me. I get:
> 
> ```
> Vocab size is 14957
> Comparing language...
> ```
> 
> But I'm not sure what this comment was for:
> 
> > Please move SE_words_GH_zscores.csv and SE_words_GH.list to src/data/

`//src:find_SE_words` generates these two files (I think it should be G instead of GH). We need to save it for classifier training.

**Pull request merged**